styles.css

```diff
@@ -114,3 +117,346 @@
   height: 20px;
   color: #505f79;
 }
+
+/* section 2 */
+
+.border_section2 {
+  width: 327px;
+  height: 552px;
+  background: #fff;
+  border-radius: 16px;
+  margin: 24px;
+  box-sizing: border-box;
+  border: 1px solid #dfe1e6;
+  margin-bottom: 88px;
+}
```

Comment:
- [ ] It is required that the main container of the projects (works) has a `display: grid` for the layout flexibility. Kindly apply `display: grid` to the main container of the projects list.